token/services/db/sql/common/init.go

```diff
@@ -30,26 +30,23 @@ type tableNames struct {
 	TokenLocks             string
 }
 
-func GetTableNames(prefix string) (tableNames, error) {
-	nc, err := common.NewTableNameCreator(prefix)
-	if err != nil {
-		return tableNames{}, err
-	}
+func GetTableNames(prefix string, params ...string) (tableNames, error) {
+	nc := db.NewTableNameCreator()
 
 	return tableNames{
-		Movements:              nc.MustGetTableName("movements"),
-		Transactions:           nc.MustGetTableName("transactions"),
-		TransactionEndorseAck:  nc.MustGetTableName("transaction_endorsements"),
-		Requests:               nc.MustGetTableName("requests"),
-		Validations:            nc.MustGetTableName("request_validations"),
-		Tokens:                 nc.MustGetTableName("tokens"),
-		Ownership:              nc.MustGetTableName("token_ownership"),
-		Certifications:         nc.MustGetTableName("token_certifications"),
-		TokenLocks:             nc.MustGetTableName("token_locks"),
-		PublicParams:           nc.MustGetTableName("public_params"),
-		Wallets:                nc.MustGetTableName("wallets"),
-		IdentityConfigurations: nc.MustGetTableName("identity_configurations"),
-		IdentityInfo:           nc.MustGetTableName("identity_information"),
-		Signers:                nc.MustGetTableName("identity_signers"),
+		Movements:              nc.MustGetTableName(prefix, "movements", params...),
```

Comment:
postgres has a limit in the name length of the tables and it's easy to get to the limit considering the tms prefix. we can find a more sustainable solution like hashing